src/apps/backend/modules/config/config_service.py

```diff
@@ -46,6 +46,14 @@ def get_token_expiry_days() -> int:
     def get_web_app_host() -> str:
         return DictUtil.required_get_str(input_dict=ConfigManager.config, key="WEB_APP_HOST")
 
+    @staticmethod
+    def get_default_otp_enabled() -> bool:
```

Comment:
Please move all OTP related config into OTP module.

If you think about this approach, and all other module start dumping their methods here, this class is going to be HUGE. Plus the job of the module is to provide infrastructure to get config values. Each module should be responsible to get their own config in their own module